src/entities/day.ts

```diff
@@ -82,15 +86,15 @@ export const fullDayMap: Partial<Record<Day, FullDayJa>> = {
   AnyTime: "随時",
 };
 
-export const weekdayNum = (day: Day): number => weekdays.indexOf(day);
+export const weekdayNum = (day: WeekDay): number => weekdays.indexOf(day);
 export const weekNum = (day: Day): number => week.indexOf(day);
 
-export const weekDayToJa = (day: Day): WeekDayJa => weekdayMap[day] ?? "月";
+// 消したい
+export const dayToWeekJa = (day: Day): WeekDayJa => weekdayMap[day] ?? "月";
 export const dayToJa = (day: Day): DayJa => weekMap[day] ?? "月";
+export const dayToFullDayja = (day: Day): FullDayJa => fullDayMap[day] ?? "月";
```

Comment:
```suggestion
// 消したい
export const dayToWeekJa = (day: Day): WeekDayJa => weekdayMap[day];
export const dayToJa = (day: Day): DayJa => weekMap[day];
export const dayToFullDayja = (day: Day): FullDayJa => fullDayMap[day];
```
なくても問題ありません。`??` でつなぐやり方は Map にない場合に問答無用で変換してしまう危険なやり方なのでこちらのほうが安全です。